microsoft-365/enterprise/manage-multitenant-org-settings.md

```diff
@@ -159,8 +159,21 @@ These permissions are required to fetch cross-tenant synchronization details and
   
   - [Synchronization.Read.All](/graph/permissions-reference#synchronizationreadall)
   
+
 This permission is required to gather details regarding the multitenant organization.
 
 - Reading MTO details 
 
   - [MultiTenantOrganization.Read.All](/graph/permissions-reference#multitenantorganizationreadall)
+
+## Manage external tag removal (Private Preview)
+
+If a tenant has enabled external tags in Outlook to help users identify content from external tenants, MTO group admins can now choose to suppress these tags for members of the multitenant organization. This setting allows for a more seamless collaboration experience and admins can enable this setting in the MAC MTO portal. 
+
+![Edit external tag setting](media/manage-multitenant-org-settings/edit-external-tag-setting.png)
```

Comment:
I would reword the setting details in this image:

Edit Outlook external tag settings
Suppress external tags in Outlook emails received from MTO members. Learn more about cross-tenant external tag removal for Outlook.

[] Suppress Outlook external tag for MTO members